docs/admin/install.rst

```diff
@@ -1405,8 +1416,8 @@ a backend looks like this:
 
    :ref:`Redis broker configuration in Celery <celery:broker-redis-configuration>`
 
-For development, you might want to use eager configuration, which does process
-all tasks in place, but this will have performance impact on Weblate:
+For development, you might want to use Eager configuration, which processes
```

Comment:
Not sure if "eager" or "Eager"

Reply:
https://docs.celeryproject.org/en/stable/userguide/testing.html

Reply:
Would that be development of Weblate, or "in production"?